src/services/homework/hooks/submissions.js

```diff
@@ -340,23 +329,23 @@ exports.before = () => ({
 		canRemoveOwner,
 		noDuplicateSubmissionForTeamMembers,
 		populateCourseGroup,
-		maxTeamMembers,
-		globalHooks.permitGroupOperation,
+		maxTeamMembersHook,
+	//	globalHooks.permitGroupOperation,
 		canGrade,
 	],
 	remove: [
-		globalHooks.hasPermission('SUBMISSIONS_CREATE'),
+		hasPermission('SUBMISSIONS_CREATE'),
 		insertSubmissionData,
 		insertHomeworkData,
 		insertSubmissionsData,
-		globalHooks.permitGroupOperation,
+	//	globalHooks.permitGroupOperation,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 tabs but found 1. (indent)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4605411)